src/components/MenuToolButton.vue

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+import { watch, ref, computed, defineComponent } from 'vue';
+import { useDisplay } from 'vuetify';
+import ToolButton from './ToolButton.vue';
+
+export default defineComponent({
+  name: 'MenuToolButton',
+  props: {
+    icon: { type: String, required: true },
+    name: { type: String, required: true },
+    size: { type: [Number, String], default: 40 },
+    active: Boolean,
+    disabled: Boolean,
+  },
+  components: {
+    ToolButton,
+  },
+  setup(props, { emit }) {
+    const display = useDisplay();
+    const showLeft = computed(() => !display.mobile.value);
+
+    const menuOn = ref(false);
+
+    // Turn off menu if tool clicked and already on
+    const handleActivatorClicked = () => {
+      menuOn.value = !menuOn.value;
+      emit('click');
+    };
+
+    // Turn off menu if tool deselected
+    const active = computed(() => props.active);
+    watch(active, (on) => {
+      if (!on) {
+        menuOn.value = false;
+      }
+    });
+
+    return { showLeft, menuOn, handleActivatorClicked };
+  },
+});
+</script>
+
+<template>
+  <v-menu
+    persistent
+    no-click-animation
+    :close-on-content-click="false"
+    v-model="menuOn"
+    :location="showLeft ? 'left' : 'right'"
```

Comment:
@aylward what's your opinion on these menus (e.g. paint menu) showing up on the left vs the right?

@PaulHax if we opt for left-opening menus, we need to make sure the menu content has enough [elevation](https://vuetifyjs.com/en/styles/elevation/). Right now, left menus blend in with the sidebar.

Reply:
Added `elevation-24` on the menu popup.